### ISSUE TITLE
Card Tile Tests Are Failing

### ISSUE THREAD

**Issue by redreceipt** [ApollosProject/apollos-prototype#662]
### Apollos Version: 0.8.2

`CardTile` snapshot is failing in `master`...

### Expected Behavior

All greens.

### Actual Behavior

Some reds. 

```
FAIL src/CardTile/CardTile.tests.js
  ● the CardTile component › should render card details with date

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot "the CardTile component should render card details with date 1".

    - Snapshot
    + Received

    @@ -297,11 +297,11 @@
                        "fontSize": 12,
                        "lineHeight": 17.28,
                      }
                    }
                  >
    -               33y
    +               34y
                  </Text>
                </View>
              </View>
            </View>
          </View>

      65 |       </Providers>
      66 |     );
    > 67 |     expect(tree).toMatchSnapshot();
         |                  ^
      68 |   });
      69 |   it('should render children', () => {
      70 |     const tree = renderer.create(

      at Object.toMatchSnapshot (src/CardTile/CardTile.tests.js:67:18)

 › 1 snapshot failed.
```

### Possible Fix

<details><summary>Additional Info</summary>

### Your Environment

- Environment name and version (e.g. Chrome 39, node.js 5.4):
- Operating System and version (desktop or mobile):
- Link to your project:

### Steps to Reproduce

1.  first...
2.
3.
4.

### Stack Trace

</details>

**Comment by redreceipt** [comment]
@vinnyjth this is the snap that's failing in all my PRs right now, turns out it's in `master`.

**Comment by vinnyjth** [comment]
Fixed in #603
